packages/@aws-cdk/aws-dynamodb-global/lib/lambda-global-dynamodb.ts

```diff
@@ -0,0 +1,63 @@
+import cfn = require("@aws-cdk/aws-cloudformation");
+import iam = require("@aws-cdk/aws-iam");
+import lambda = require("@aws-cdk/aws-lambda");
+import cdk = require("@aws-cdk/cdk");
+import path = require("path");
+import { GlobalTableProps } from "./aws-dynamodb-global";
+
+/**
+ * A stack that will make a Lambda that will launch a lambda to glue
+ * together all the DynamoDB tables into a global table
+ */
+export class LambdaGlobalDynamoDBMaker extends cdk.Stack {
```

Comment:
Sorry for nitpicking, I am a bit sensitive to naming. Can you rename this to `GlobalTableCoordinator` (it's an internal class, I know...).